examples/opentelemetry/Cargo.toml

```diff
@@ -9,9 +9,9 @@ license = "MIT/Apache-2.0"
 [dependencies]
 actix-web = "4"
 tracing = "0.1.19"
-opentelemetry = { version = "0.19", features = ["rt-tokio-current-thread"] }
-opentelemetry-jaeger = { version = "0.18", features = ["rt-tokio-current-thread"] }
-tracing-opentelemetry = { version = "0.19" }
+opentelemetry = { version = "0.22", features = ["rt-tokio-current-thread"] }
+opentelemetry-jaeger = { version = "0.21", features = ["rt-tokio-current-thread"] }
+tracing-opentelemetry = { version = "0.23" }
 tracing-subscriber = { version = "0.3", features = ["registry", "env-filter"] }
 tracing-bunyan-formatter = "0.3"
 tracing-actix-web = { path = "../..", features = ["opentelemetry_0_19"] }
```

Comment:
you should also update this feature, that's likely the source of the build failures